host-configs/llnl/blueos_3_ppc64le_ib_p9/clang@upstream_link_with_nvcc.cmake

```diff
@@ -48,26 +50,24 @@ set(BLT_MPI_COMMAND_APPEND "mpibind" CACHE PATH "")
 set(ENABLE_OPENMP ON CACHE BOOL "")
 
 # Override default link flags because linking with nvcc
-set(OMP_HOME ${CLANG_HOME}/ibm/omprtl)
+set(OMP_HOME ${CLANG_HOME}/release)
```

Comment:
This path changed in the recent drop, apparently.
I tested the new one against the internal tests and the ``calc_pi`` tutorial.